app/src/main/res/layout-sw600dp/story_fragment.xml

```diff
@@ -71,9 +62,9 @@
         android:background="@color/component_color_story_activity_background_color"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        android:paddingBottom="@dimen/story_fragment_padding_bottom"
         android:paddingStart="64dp"
         android:paddingEnd="64dp"
-        android:paddingBottom="@dimen/story_fragment_padding_bottom"
```

Comment:
Note: I'll revert this extra change (since it's non-functional, and it's better to avoid churn in PRs we intend to cherry-pick) when I have a reason to push a new commit (so that the existing CI can run fully).

Reply:
This has been done in the latest changes.